net-misc/bsdwhois/files/bsdwhois-1.43.2.1-pull-in-GNU_SOURCE.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/configure.in b/configure.in
+index 7de149e..fb57dda 100644
+--- a/configure.in
++++ b/configure.in
+@@ -8,6 +8,8 @@ AM_INIT_AUTOMAKE()
+ 
+ # Checks for programs.
+ AC_PROG_CC
++# pulls in vasprintf, get GNU_SOURCE
++AC_USE_SYSTEM_EXTENSIONS
```

Comment:
looks right

Reply:
no and no. I will put it into the ebuild as a flag instead, shall I?